### PR TITLE
Split LDAP Kerberos bucket in 2 projects for slow Windows

### PR DESCRIPTION
**Pull request by kristip17**
Split the bucket so FULL mode FAT can complete on slow Windows runs. I suspect running the ApacheDS KDC/LDAP with the test is making generally slow Windows runs extra slow for this bucket.

I added an `...ldap_fat.krb5.2` project and split the `FatSuite` between the two projects. After testing, the LITE bucket timed out once, so I moved a LITE test over to the new project as well.

RTC 286705

### CODE REVIEW

**Comment by kristip17** [comment]
#build

**Review by jvanhill — CHANGES_REQUESTED** [review]
You need to update the ../.github/test-categories/SECURITY_4 file to add the new bucket. I also would prefer if you named the original bucket with .1 as well.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.2/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	org.apache.directory.server:apacheds-all;version=latest, \
+	org.apache.directory.server:kerberos-client;version=latest, \
+	com.ibm.ws.security.registry;version=latest,\
+	com.ibm.websphere.security.wim.base;verison=latest, \
+	com.ibm.ws.security.wim.adapter.ldap;verison=latest, \
+	com.ibm.ws.security.registry_test.servlet;version=latest, \
+	com.unboundid:unboundid-ldapsdk;version=5.1.0,\
+	com.ibm.ws.com.unboundid;version=latest, \
+	org.slf4j:slf4j-jdk14;version=latest, \
+	com.ibm.ws.security.wim.adapter.ldap_fat.krb5;version=latest
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.2/bnd.bnd]:
```suggestion
	com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1;version=latest
```

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.2/build.gradle [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+def appBuildDir = "${buildDir}/test-application"
+
+dependencies { 
+    requiredLibs 'org.apache.directory.server:apacheds-all:2.0.0-M24',
+	'org.apache.directory.server:kerberos-client:2.0.0-M24',
+    project(':fattest.simplicity'),
+    project(':com.ibm.ws.security.registry_test.servlet'),
+    project(':com.ibm.ws.com.unboundid'),
+    project(':com.ibm.ws.security.wim.adapter.ldap_fat.krb5'),
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.2/build.gradle]:
```suggestion
    project(':com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1'),
```

**Review by jvanhill — CHANGES_REQUESTED** [review]
Two updates to the project name in build files.

**Review by jvanhill — APPROVED** [review]

**Comment by kristip17** [comment]
#libby

**Pull request merged**